projects/portal/src/app/views/staking/create-validator/create-validator.component.html

```diff
@@ -245,11 +206,11 @@ <h4 class="pt-3">Gas Settings</h4>
             type="number"
             [min]="0"
             required
-            readonly
```

Comment:
This component is not for general user. So, in this component, this settings should be readonly.

Reply:
In this component, expected behavior is all form can't be edited.